docker/web_nginx.conf

```
@@ -12,22 +12,29 @@ map $request_uri$args $is_sus_random_sort {
     "~^/authors/.*sort=random_" 1;
 }
 
-# Check empty referer
+# 1. Detect empty referer
 map $http_referer $is_empty_referer {
     default 0;
     "" 1;
 }
 
+# 2. Check if session is logged out
+map $cookie_session $is_logged_out {
+    default 0;
+    "" 1;  # session cookie is empty
+}
+
+# 3. Check if path requires referer or login
 map $request_uri$args $requires_referer {
     default 0;
     "~*^/(qrcode|admin|wp-login|show-records)" 1;
-    "~*/edit" 1;
+    "~*^/edit(/|$|\?)" 1;
```

We don't want the `^` here otherwise only `openlibrary.org/edit` will match .
```suggestion
    "~*/edit(/|$|\?)" 1;
```